src/clr-core/styles/layout/_optimize.scss

```diff
@@ -6,15 +6,15 @@
 // a "minified" value to reduce overall bundle size.
 
 :root {
-  --cdlsxs: var(--cds-token-layout-space-xs);
-  --cdlssm: var(--cds-token-layout-space-sm);
-  --cdlsmd: var(--cds-token-layout-space-md);
-  --cdlslg: var(--cds-token-layout-space-lg);
-  --cdlsxl: var(--cds-token-layout-space-xl);
+  --clxs: var(--cds-token-layout-space-xs, #{$cds-token-layout-space-xs-static});
+  --clsm: var(--cds-token-layout-space-sm, #{$cds-token-layout-space-sm-static});
+  --clmd: var(--cds-token-layout-space-md, #{$cds-token-layout-space-md-static});
+  --cllg: var(--cds-token-layout-space-lg, #{$cds-token-layout-space-lg-static});
+  --clxl: var(--cds-token-layout-space-xl, #{$cds-token-layout-space-xl-static});
 }
 
-$cds-token-layout-space-xs: var(--cdlsxs, $cds-token-layout-space-xs-static);
-$cds-token-layout-space-sm: var(--cdlssm, $cds-token-layout-space-sm-static);
-$cds-token-layout-space-md: var(--cdlsmd, $cds-token-layout-space-md-static);
-$cds-token-layout-space-lg: var(--cdlslg, $cds-token-layout-space-lg-static);
-$cds-token-layout-space-xl: var(--cdlsxl, $cds-token-layout-space-xl-static);
+$cds-token-layout-space-xs: var(--clxs);
```

Comment:
By inverting the fallback we only have to define the fallback once instead for every single reference to the sass variables.